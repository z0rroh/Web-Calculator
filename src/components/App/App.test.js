import React from 'react'
import { shallow } from 'enzyme'
import App from './App'
import Calculator from '../Calculator/Calculator'

describe('<App/>', () => {

   let wrapp;
   beforeEach( () => wrapp = shallow( <App/> ) )

   it('Should render correctly', () => {
      expect(wrapp).toMatchSnapshot()
   })

   it('Should render a div', () => {
      expect(wrapp.find('div').length).toEqual(1);
   })

   it('Should render to Calculator component', () => {
      expect(wrapp.containsMatchingElement(<Calculator />)).toEqual(true)
   })
})