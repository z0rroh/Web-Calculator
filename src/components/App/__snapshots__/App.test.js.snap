// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> Should render correctly 1`] = `
<div
  className="app-container"
>
  <Calculator />
</div>
`;
